test/integration/select_test.lua

```diff
@@ -77,6 +78,54 @@ pgroup.test_not_valid_value_type = function(g)
     t.assert_str_contains(err.err, "Supplied key type of part 0 does not match index part type: expected unsigned")
 end
 
+pgroup.test_select_nil = function(g)
+    local space = 'customers'
+    local crit_log = "C> empty or nil `select` call on user space=" .. space
+    local valid_condition = {'>=', 'id', 2}
+    local tests = {
+        {crit = true, err = false, uc = nil, opts = nil},
+        {crit = true, err = false, uc = nil, opts = {}},
+        {crit = true, err = false, uc = nil, opts = {fullscan = false}},
+        {crit = true, err = false, uc = {}, opts = nil},
+        {crit = true, err = false, uc = {}, opts = {}},
+        {crit = true, err = false, uc = {}, opts = {fullscan = false}},
```

Comment:
Why are there no `crit = true, error = true` cases? It is hard to implement?

Reply:
The tests have been rewritten.